pkg/resources/managed_account.go

```diff
@@ -128,6 +128,12 @@ func ReadManagedAccount(d *schema.ResourceData, meta interface{}) error {
 	stmt := snowflake.ManagedAccount(id).Show()
 	row := snowflake.QueryRow(db, stmt)
 	a, err := snowflake.ScanManagedAccount(row)
+
+	if err == sql.ErrNoRows {
+		log.Printf("[WARN] managed account (%s) not found, removing from state file", d.Id())
```

Comment:
I think it will only be removed form the state file if running a refresh or apply operation, so the log message should avoid suggesting that it is being remove durably.

Reply:
The level is fine, I just think that it would be more clear if we didn't state that it was removed from the state file, since it might only be removed from the transient, in-memory state used for a plan.